test/ex_pesa/Jenga/kyc_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule ExPesa.Jenga.KYCTest do
+  @moduledoc false
```

Comment:
move this file to a directory `kyc` to align with structuring system

Reply:
It's a single file and there's no other endpoint for KYC as opposed to others like sending money. I don't see a reason to create a folder just for that file